weaver/wps_restapi/processes/processes.py

```diff
@@ -22,7 +23,8 @@
     STATUS_SUCCEEDED,
 )
 from weaver.store.base import StoreServices, StoreProcesses, StoreJobs
-from weaver.utils import get_any_id, get_any_value, get_settings, raise_on_xml_exception, wait_secs, get_cookie_headers
+from weaver.utils import get_any_id, get_any_value, get_settings, raise_on_xml_exception, wait_secs, get_cookie_headers, \
+    get_registry
```

Comment:
too long change to (newlines as required)
```
from weaver.utils import (
    get_any_id, get_any_value, get_settings, raise_on_xml_exception, 
    wait_secs, get_cookie_headers, get_registry
)
```

Reply:
maybe we should have a linter running on the ci...